src/WebJobs.Script/Rpc/WebHostLanguageWorkerChannelManager.cs

```diff
@@ -55,8 +55,9 @@ private async Task<ILanguageWorkerChannel> InitializeLanguageWorkerChannel(strin
             try
             {
                 languageWorkerChannel = _languageWorkerChannelFactory.CreateLanguageWorkerChannel(scriptRootPath, runtime, null, 0);
-                await languageWorkerChannel.StartWorkerProcessAsync();
                 AddOrUpdateWorkerChannels(runtime, languageWorkerChannel);
```

Comment:
Can we just pass in the channel ID and change this to be called something like "AddWorkerChannel"?

Reply:
This is still add or update as this method is for workerChannels maintained by the webhost channel manager